material_theme_builder/README.md

```diff
@@ -0,0 +1,20 @@
+# Build a Material Theme
```

Comment:
We have a general format for these READMEs that we like to use (goals, important bits, etc.). You can see it in action here:

https://github.com/flutter/samples/tree/master/isolate_example

Reply:
Done.

Reply:
Please re review that file.